modules/ocf_rancid/manifests/init.pp

```diff
@@ -49,4 +40,35 @@
     special => 'hourly',
     require => Package['rancid'],
   }
+
+  # Delete all logs older than 14 days.
+  # This would be better with logrotate, but the log files contain the date in
+  # their name, which logrotate doesn't handle very well. The files aren't large
+  # at all, so they don't need to be compressed, there's just a lot of them
+  # since they are created once for every hour.
+  cron { 'clean-rancid-logs':
+    command => 'find /var/log/rancid/* -mtime +14 > /dev/null',
+    user    => 'rancid',
+    special => 'hourly',
+    require => Package['rancid'],
+  }
+
+  # Don't push to GitHub for dev-* hosts to prevent duplicate backups with
+  # different commit hashes
+  if $::hostname !~ /^dev-/ {
```

Comment:
we should really turn this into a global/hiera variable rather than repeating this dev- hostname junk everywhere

maybe a good starter ticket

Reply:
Good idea, I like the idea of making this a starter ticket because it wouldn't be too hard and would touch a few different parts of puppet, so that would be cool